src/components/Chart.jsx

```diff
@@ -113,7 +114,7 @@ class Chart extends Component {
                                         }
                                     </RenderInsideChart>
                                     <RenderInsideChart at="subholder">
-                                        {children}
+                                        {isChartReady && children}
```

Comment:
`RenderInsideChart` should and does take care of it's children. 
Only when the chart is ready we should render them, if that's not the case it needs to be fixed in `RenderInsideChart`.